plugins/keyboard-navigation/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "blockly-plugin-keyboard-navigation",
+  "version": "0.20201029.0",
```

Comment:
I thought we had fixed this in the script already but it should be 0.1.0.

Reply:
done